docs/test_resource_yamls/README.md

```diff
@@ -8,11 +8,8 @@ The files are named according to the endpoint they were used to test. Some may c
 
 - `configmap.yaml`
 - `daemonset.yaml`
-- `deployment.yaml`
```

Comment:
why?

Reply:
I removed files that are no longer needed. Installing the Helm chart now creates the ingress, deployment, and service automatically.